### DISCUSSION TITLE
cant see my devices

### DISCUSSION THREAD

**Discussion by andrewsameh1994** [nVuln/homebridge-lg-thinq#180] [General]
hi i added my account to the plugin but i cant see my device (tv) 
what should i don in the add devices stage? i should type TC and give it a random id?

**Comment by nVuln** [comment]
TV is not supported by this plugin, it's webos
please try use this plugin: https://github.com/merdok/homebridge-webos-tv

**Reply by andrewsameh1994** [reply]
Thanks alot for the elaboration.
will check it out.

i just thought all devices in thinq will be imported to homebridge
